package.json

```diff
@@ -8,6 +8,7 @@
   "typings": "typings/glamorous.d.ts",
   "scripts": {
     "start": "nps",
+    "doctoc": "doctoc --maxlevel 3 --title \"**Table of Contents**\"",
```

Comment:
Sorry to make you do all that `package-scripts.js` work only to move it back @PeterDaveHello. Things weren't working too well with the `nps` + `lint-staged` integration. I'll be working on that :)